### DISCUSSION TITLE
Hedera Archive node requirements

### DISCUSSION THREAD

**Discussion by SyedMuhamadYasir** [hiero-ledger/hiero-mirror-node#7639] [Help]
Hi, i am looking to set up a Hedera Archive node, which i presume in hedera lingo, is called the Mirror Node

I want the entire transaction history of the hedera mainnet.

Therefore, please communicate what hardware is required for an archive node ?

**Comment by steven-sheehy** [comment]
There are some hardware recommendation on our [docs](https://docs.hedera.com/hedera/core-concepts/mirror-nodes/run-your-own-beta-mirror-node) site. If you want full history it's at 36 TB so you'll need to adjust the db disk accordingly.
